docker/Dockerfile

```diff
@@ -124,19 +140,29 @@ RUN echo "pip install pip \\" >> $WORKSPACE/.install-dependencies.sh && \
     fi && \
     echo ";" >> $WORKSPACE/.install-dependencies.sh
 
-# add custom installation commands to install script
-ARG CUSTOM_SCRIPT_FILE="docker/custom.sh"
```

Comment:
Should we rename `CUSTOM_SCRIPT_FILE`?

Reply:
I would tend to keep the naming of the `custom.sh`